scripts/template_upload_scvmm.py

```diff
@@ -201,16 +202,14 @@ def run(**kwargs):
                     client,
                     host_fqdn,
                     new_template_name,
-                    use_library,
-                    use_network,
-                    use_os_type,
+                    library,
+                    network,
+                    os_type,
                     username_scvmm,
                     cores,
                     ram
                 )
             try:
-                wait_for(check_template_exists,
-                         [client, new_template_name], fail_condition=False, delay=5)
```

Comment:
Reason for dropping this wait_for? Doesn't look like there's any assertion that `make template` actually worked.